src/main/resources/standardTestCases/defined_function/defined_function.wdl

```diff
@@ -0,0 +1,50 @@
+task defined_in_task {
+  File? f
+  Boolean is_defined = defined(f)
+  command {
+    ${true="cat" false="echo no file" is_defined} ${f}
+  }
+  runtime {
+    docker: "ubuntu:latest"
+  }
+  output {
+    String out = read_string(stdout())
+  }
+}
+
+task cat {
+  File f
+  command { cat ${f} }
+  runtime { docker: "ubuntu:latest" }
+  output { String out = read_string(stdout()) }
+}
+
+task mk_file {
+  Int index
+  command { echo "file_${index}" > out }
+  runtime { docker: "ubuntu:latest" }
+  output { File f = "out" }
+}
+
+workflow defined_function {
+  Array[Boolean] mask = [true, false, true, false]
+  Array[Int] indices = range(4)
+  Array[Pair[Boolean, Int]] masked_indices = zip(mask, indices)
+
+  scatter (p in masked_indices) {
+    if (p.left) {
+      Int good_index = p.right
+      call mk_file { input: index = good_index }
+    }
+    # call defined_in_task { input: f = mk_file.f }
```

Comment:
Hmm, I half agree. I really like that it's easy to re-enable to verify a fix for https://github.com/broadinstitute/cromwell/issues/1982... but yeah I don't like the commenting